go.mod

```diff
@@ -19,17 +20,14 @@ require (
 	github.com/stretchr/testify v1.3.0
 	golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c // indirect
 	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a // indirect
+	golang.org/x/sys v0.0.0-20190312061237-fead79001313 // indirect
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
-	golang.org/x/tools v0.0.0-20190328211700-ab21143f2384 // indirect
-	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.2.2
 	k8s.io/api v0.0.0-20181213150558-05914d821849
-	k8s.io/apimachinery v0.0.0-20181127105237-2b1284ed4c93
+	k8s.io/apimachinery v0.0.0-20181127025237-2b1284ed4c93
 	k8s.io/client-go v0.0.0-20181213151034-8d9ed539ba31
-	k8s.io/code-generator v0.0.0-20181116203124-c2090bec4d9b // indirect
```

Comment:
I'm surprised this dependency is going away.

Reply:
I just did a `go mod tidy`

Reply:
If you want to manage your tools with go modules, take a look to https://github.com/go-modules-by-example/index/blob/master/010_tools/README.md

Reply:
I can create and add a `tools.go` as suggested in the article if your want?

Reply:
Ohh, that's how you do it. 👍  `tools.go` sounds like a great idea.

Reply:
Note that the hash c2090bec4d9b doesn't exist:
https://github.com/kubernetes/code-generator/search?q=c2090bec4d9b&type=Commits

Reply:
the date doesn't exist also